mucgpt-frontend/src/components/History/History.module.css

```
@@ -83,15 +87,40 @@
+    flex-direction: column;
+    align-items: stretch;
+    gap: 8px;
+    width: 100%;
+    margin-bottom: 8px;
+}
+
+.historyActions > * {
+    width: 100%;
+}
+
+.historyActions :global(.fui-Button) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)